shared/constants/types/fs.js

```diff
@@ -326,3 +328,1202 @@ export type FavoriteFolder = {
     can_self_help: boolean,
   },
 }
+
+export type MediaType = 'image' | 'video' | 'audio'
+
+// export type FileType = MediaType | 'text' | 'unknown'
+export type FileType = 'text' | 'unknown'
+
+// TODO: integrate with mime types after jzila's PR is merged.
+export const inferFileTypeFromName = (name: string): FileType => {
```

Comment:
You shouldn't need this list.

https://github.com/keybase/client/blob/88484c9835dbac1be7c6bc81854e97ad88c85ddd/shared/constants/fs.js#L442

and

https://github.com/keybase/client/blob/88484c9835dbac1be7c6bc81854e97ad88c85ddd/shared/constants/fs.js#L290-L299

Reply:
Oops sorry, missed that comment.

Reply:
Hmm it doesn't have many programming language suffix though

Reply:
On the other hand, it's impossible to cover all extensions, not mention files with no extensions that are also text. So missing `.go` and `.py` is the same in this sense. Perhaps we can provide a button in the default view to "View as Text".